include/net/fota_download.h

```diff
@@ -146,6 +146,28 @@ int fota_download(const char *host, const char *file, const int *sec_tag_list,
 		  uint8_t sec_tag_count, uint8_t pdn_id, size_t fragment_size,
 		  const enum dfu_target_image_type expected_type);
 
+/**@brief Download the given file with the specified image type from the given host.
+ *
+ * Validate that the file type matches the expected type before proceeding with the download.
+ * When the download is complete, the secondary slot of MCUboot is tagged as having
+ * valid firmware inside it. The completion is reported through an event.
+ *
+ * URI parameters (host and file) are not copied, so pointers must stay valid
+ * until download is finished.
```

Comment:
```suggestion
 * URI parameters (host and file) and sec_tag lists are not copied, so pointers must stay valid
 * until download is finished.
```

Reply:
It has already been documented [here ](https://github.com/nrfconnect/sdk-nrf/blob/main/include/net/download_client.h#L105)but I would suggest to change it to be `const int sec_tag_list[NRF_SOCKET_TLS_MAX_SEC_TAG_LIST_SIZE];`, i.e. max 7 integers